+ source ../common/common_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Thu Dec 29 02:31:27 2022 - INFO  - Start environmental preparation.
+ DNF_INSTALL openvswitch
+ pkgs=openvswitch
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs openvswitch --node 1 --tempfile ''
+ tmpfile2=/tmp/tmpjfs8hwgx
+ '[' -z '' ']'
+ tmpfile=/tmp/tmpjfs8hwgx
+ service=ovs-vswitchd.service
++ date '+%Y-%m-%d %T'
+ log_time='2022-12-29 02:32:52'
+ flag=false
++ grep Enforcing
++ getenforce
+ '[' ']'
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Thu Dec 29 02:32:53 2022 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Thu Dec 29 02:32:53 2022 - INFO  - Start testing...
+ service ovs-vswitchd.service restart
oe_test_service_ovs-vswitchd.sh: line 37: service: command not found
+ CHECK_RESULT 127 0 0 'ovs-vswitchd.service restart failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='ovs-vswitchd.service restart failed'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'ovs-vswitchd.service restart failed'
+ LOG_ERROR 'ovs-vswitchd.service restart failed'
+ message='ovs-vswitchd.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'ovs-vswitchd.service restart failed'
Thu Dec 29 02:32:54 2022 - ERROR - ovs-vswitchd.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_ovs-vswitchd.sh line 38'
+ message='oe_test_service_ovs-vswitchd.sh line 38'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_ovs-vswitchd.sh line 38'
Thu Dec 29 02:32:54 2022 - ERROR - oe_test_service_ovs-vswitchd.sh line 38
+ return 0
+ service ovs-vswitchd.service stop
oe_test_service_ovs-vswitchd.sh: line 39: service: command not found
+ CHECK_RESULT 127 0 0 'ovs-vswitchd.service stop failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='ovs-vswitchd.service stop failed'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'ovs-vswitchd.service stop failed'
+ LOG_ERROR 'ovs-vswitchd.service stop failed'
+ message='ovs-vswitchd.service stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'ovs-vswitchd.service stop failed'
Thu Dec 29 02:32:55 2022 - ERROR - ovs-vswitchd.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_ovs-vswitchd.sh line 40'
+ message='oe_test_service_ovs-vswitchd.sh line 40'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_ovs-vswitchd.sh line 40'
Thu Dec 29 02:32:56 2022 - ERROR - oe_test_service_ovs-vswitchd.sh line 40
+ return 0
+ service ovs-vswitchd.service start
oe_test_service_ovs-vswitchd.sh: line 41: service: command not found
+ CHECK_RESULT 127 0 0 'ovs-vswitchd.service start failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='ovs-vswitchd.service start failed'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'ovs-vswitchd.service start failed'
+ LOG_ERROR 'ovs-vswitchd.service start failed'
+ message='ovs-vswitchd.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'ovs-vswitchd.service start failed'
Thu Dec 29 02:32:56 2022 - ERROR - ovs-vswitchd.service start failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_ovs-vswitchd.sh line 42'
+ message='oe_test_service_ovs-vswitchd.sh line 42'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_ovs-vswitchd.sh line 42'
Thu Dec 29 02:32:57 2022 - ERROR - oe_test_service_ovs-vswitchd.sh line 42
+ return 0
+ service ovs-vswitchd.service status
+ grep 'Active: active (running)'
oe_test_service_ovs-vswitchd.sh: line 43: service: command not found
+ CHECK_RESULT 1 0 0 'ovs-vswitchd.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='ovs-vswitchd.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'ovs-vswitchd.service start failed'
+ LOG_ERROR 'ovs-vswitchd.service start failed'
+ message='ovs-vswitchd.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'ovs-vswitchd.service start failed'
Thu Dec 29 02:32:58 2022 - ERROR - ovs-vswitchd.service start failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_ovs-vswitchd.sh line 44'
+ message='oe_test_service_ovs-vswitchd.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_ovs-vswitchd.sh line 44'
Thu Dec 29 02:32:58 2022 - ERROR - oe_test_service_ovs-vswitchd.sh line 44
+ return 0
+ grep -i 'fail\|error'
+ journalctl --since '2022-12-29 02:32:52' -u ovs-vswitchd.service
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ CHECK_RESULT 1 1 0 'There is an error message for the log of ovs-vswitchd.service'
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log='There is an error message for the log of ovs-vswitchd.service'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ service ovs-vswitchd.service start
oe_test_service_ovs-vswitchd.sh: line 47: service: command not found
+ sed -i 's\ExecStart=/usr/share/openvswitch/scripts/ovs-ctl\ExecStart=/usr/share/openvswitch/scripts/ovs-ctl --no-mlockall\g' /usr/lib/systemd/system/ovs-vswitchd.service
+ systemctl daemon-reload
+ service ovs-vswitchd.service reload
oe_test_service_ovs-vswitchd.sh: line 50: service: command not found
+ CHECK_RESULT 127 0 0 'ovs-vswitchd.service reload failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='ovs-vswitchd.service reload failed'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'ovs-vswitchd.service reload failed'
+ LOG_ERROR 'ovs-vswitchd.service reload failed'
+ message='ovs-vswitchd.service reload failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'ovs-vswitchd.service reload failed'
Thu Dec 29 02:33:03 2022 - ERROR - ovs-vswitchd.service reload failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_ovs-vswitchd.sh line 51'
+ message='oe_test_service_ovs-vswitchd.sh line 51'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_ovs-vswitchd.sh line 51'
Thu Dec 29 02:33:04 2022 - ERROR - oe_test_service_ovs-vswitchd.sh line 51
+ return 0
+ service ovs-vswitchd.service status
+ grep 'Active: active'
oe_test_service_ovs-vswitchd.sh: line 52: service: command not found
+ CHECK_RESULT 1 0 0 'ovs-vswitchd.service reload causes the service status to change'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='ovs-vswitchd.service reload causes the service status to change'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'ovs-vswitchd.service reload causes the service status to change'
+ LOG_ERROR 'ovs-vswitchd.service reload causes the service status to change'
+ message='ovs-vswitchd.service reload causes the service status to change'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'ovs-vswitchd.service reload causes the service status to change'
Thu Dec 29 02:33:04 2022 - ERROR - ovs-vswitchd.service reload causes the service status to change
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_ovs-vswitchd.sh line 53'
+ message='oe_test_service_ovs-vswitchd.sh line 53'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_ovs-vswitchd.sh line 53'
Thu Dec 29 02:33:05 2022 - ERROR - oe_test_service_ovs-vswitchd.sh line 53
+ return 0
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Thu Dec 29 02:33:05 2022 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 6
+ test 6 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu Dec 29 02:33:06 2022 - ERROR - failed to execute the case.
+ exit 6
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Thu Dec 29 02:33:07 2022 - INFO  - start environment cleanup.
+ sed -i 's\ExecStart=/usr/share/openvswitch/scripts/ovs-ctl --no-mlockall\ExecStart=/usr/share/openvswitch/scripts/ovs-ctl\g' /usr/lib/systemd/system/ovs-vswitchd.service
+ systemctl daemon-reload
+ service ovs-vswitchd.service reload
oe_test_service_ovs-vswitchd.sh: line 61: service: command not found
+ service ovs-vswitchd.service stop
oe_test_service_ovs-vswitchd.sh: line 62: service: command not found
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmpjfs8hwgx ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmpjfs8hwgx
+ '[' 0 -ne 0 ']'
+ '[' false = true ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Thu Dec 29 02:34:08 2022 - INFO  - Finish environment cleanup!
